src/test/java/com/google/sps/SetUpObject.java

```diff
@@ -0,0 +1,31 @@
+package com.google.sps.servlets;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.StringWriter;
+import java.util.*;
+import org.mockito.Mockito;
+
+public class SetUpObject {
```

Comment:
The name should be more specific about the object type. This could be any object.